Index: cz/spsmb/b3i/w08/Cviceni/StringToBinary.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package cz.spsmb.b3i.w08.Cviceni;\r\n//Napište program, který načítá řetězce složené z číslic. Pokud bude\r\n//první číslicí 0, je zbytek řetězce považovaný za osmičkové číslo.\r\n//Jsou-li na začátku znaky 0x (nebo 0X), pak se jedná o šestnáctkové\r\n// číslo. Všechna ostatní čísla jsou desítková čísla. Toto načtené číslo\r\n// vypište binárně\r\npublic class StringToBinary {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cz/spsmb/b3i/w08/Cviceni/StringToBinary.java b/cz/spsmb/b3i/w08/Cviceni/StringToBinary.java
--- a/cz/spsmb/b3i/w08/Cviceni/StringToBinary.java	
+++ b/cz/spsmb/b3i/w08/Cviceni/StringToBinary.java	
@@ -1,8 +1,18 @@
 package cz.spsmb.b3i.w08.Cviceni;
+
+import java.util.Scanner;
+
 //Napište program, který načítá řetězce složené z číslic. Pokud bude
 //první číslicí 0, je zbytek řetězce považovaný za osmičkové číslo.
 //Jsou-li na začátku znaky 0x (nebo 0X), pak se jedná o šestnáctkové
 // číslo. Všechna ostatní čísla jsou desítková čísla. Toto načtené číslo
 // vypište binárně
-public class StringToBinary {
-}
+//public class StringToBinary {
+//    public static void main(String[] args) {
+//        Scanner sc = new Scanner(System.in);
+//        int a = sc.nextInt();
+//        if (a==0){
+//            System.out.format()
+//        }
+//    }
+//}
Index: cz/spsmb/b3i/w08/Cviceni/TiskPi.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package cz.spsmb.b3i.w08.Cviceni;\r\n//Napište metodu void tiskPI(int pocetDesMist), která bude tisknout číslo\r\n//Math.PI s přesností na zadaný počet desetinných míst\r\npublic class TiskPi {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cz/spsmb/b3i/w08/Cviceni/TiskPi.java b/cz/spsmb/b3i/w08/Cviceni/TiskPi.java
--- a/cz/spsmb/b3i/w08/Cviceni/TiskPi.java	
+++ b/cz/spsmb/b3i/w08/Cviceni/TiskPi.java	
@@ -2,4 +2,7 @@
 //Napište metodu void tiskPI(int pocetDesMist), která bude tisknout číslo
 //Math.PI s přesností na zadaný počet desetinných míst
 public class TiskPi {
+    private static void int pocetDesMist() {
+
+    }
 }
