Index: cz/spsmb/b3i/w08/Cviceni/StringToLUX.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package cz.spsmb.b3i.w08.Cviceni;\r\n//Napište program, který přečte řetězec a v závislosti na jeho posledním znaku provede:\r\n//l (L) - převod řetězce na malá písmena (lower)\r\n//u (U) - převod řetězce na velká písmena (upper)\r\n//x (X) - prohození malých a velkých písmen (exchange)\r\n//Změny proveďte v řetězci, nikoliv jen na výstupu\r\npublic class StringToLUX {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cz/spsmb/b3i/w08/Cviceni/StringToLUX.java b/cz/spsmb/b3i/w08/Cviceni/StringToLUX.java
--- a/cz/spsmb/b3i/w08/Cviceni/StringToLUX.java	
+++ b/cz/spsmb/b3i/w08/Cviceni/StringToLUX.java	
@@ -1,8 +1,16 @@
 package cz.spsmb.b3i.w08.Cviceni;
+
+import java.util.Scanner;
+
 //Napište program, který přečte řetězec a v závislosti na jeho posledním znaku provede:
 //l (L) - převod řetězce na malá písmena (lower)
 //u (U) - převod řetězce na velká písmena (upper)
 //x (X) - prohození malých a velkých písmen (exchange)
 //Změny proveďte v řetězci, nikoliv jen na výstupu
 public class StringToLUX {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String a = sc.nextLine();
+
+    }
 }
