Index: cz/spsmb/b3i/w08/Character_.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package cz.spsmb.b3i.w08;\r\n\r\n//Třída Character - obalovací třída primitivního datového typu char\r\npublic class Character_ {\r\n    public static void main(String[] args) {\r\n        System.out.println(Character.isDigit('1')); //true\r\n        //Java používá Unicode kódování. Tzn., že např. znaky s hodnotami\r\n        //'\\u0BE7' - '\\u0BEF' jasou platné tamilské číslice, tudíž následující\r\n        // příklad vrátí true\r\n        System.out.format(\"znak %c, %b%n\",'\\u0BE7', Character.isDigit('\\u0BE7')); //true\r\n        System.out.println(Character.isLetter('A')); //true\r\n        System.out.println(Character.isLetterOrDigit('?')); //false\r\n        System.out.println(Character.isLowerCase('b')); //true\r\n        System.out.println(Character.isUpperCase('B')); //true\r\n        System.out.println(Character.isWhitespace('\\n')); //true\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cz/spsmb/b3i/w08/Character_.java b/cz/spsmb/b3i/w08/Character_.java
--- a/cz/spsmb/b3i/w08/Character_.java	
+++ b/cz/spsmb/b3i/w08/Character_.java	
@@ -7,6 +7,8 @@
         //Java používá Unicode kódování. Tzn., že např. znaky s hodnotami
         //'\u0BE7' - '\u0BEF' jasou platné tamilské číslice, tudíž následující
         // příklad vrátí true
+        for (char ch = '\u0BE7'; ch<='\u0BE7'; ch++ ){
+            System.out.println(ch);}
         System.out.format("znak %c, %b%n",'\u0BE7', Character.isDigit('\u0BE7')); //true
         System.out.println(Character.isLetter('A')); //true
         System.out.println(Character.isLetterOrDigit('?')); //false
